lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -50,8 +50,9 @@
             restore_non_probabilistic_dimensions)
 from improver.utilities.cube_manipulation import (
     concatenate_cubes, enforce_coordinate_ordering)
-from improver.utilities.cube_checker import (find_percentile_coordinate,
```

Comment:
All the changes in this file are to due with the promotion of time coordinates to be dimensional coordinates.

Reply:
Excellent!  We shouldn't have random dimensional coordinates with only one value...